rasa/test.py

```diff
@@ -27,6 +32,20 @@ def test_core_models_in_directory(model_directory: Text, stories: Text, output:
     plot_core_results(output, number_of_stories)
 
 
+def _test_core_models_in_directory_input(model_directory: Text) -> Text:
+    import rasa.model
+
+    p = Path(model_directory)
```

Comment:
I'd say this is optional because the function is very short, but in general it makes sense to give variables meaningful names, e.g.
```suggestion
    path_to_model_directory = Path(model_directory)
```

In this case it's really an edge case to be fair.